From: Jan Engelhardt <jengelh@inai.de>
Date: 2015-12-08 13:49:00.250213202 +0100

dxflib by default does not build a shared library, and does not
track its ABI either. So we have to do that.
---
 dxflib.pro |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

Index: dxflib-3.26.4-src/dxflib.pro
===================================================================
--- dxflib-3.26.4-src.orig/dxflib.pro
+++ dxflib-3.26.4-src/dxflib.pro
@@ -23,9 +23,9 @@ SOURCES = \
     src/dl_dxf.cpp \
     src/dl_writer_ascii.cpp
 
-TARGET = dxflib
+TARGET = dxflib-3.26.4
 TEMPLATE = lib
-CONFIG += staticlib
+CONFIG += sharedlib
 CONFIG -= qt
 DEFINES += DXFLIB_LIBRARY
 RC_FILE = dxflib.rc
